project/AppDependencies.scala

```diff
@@ -2,11 +2,11 @@ import sbt._
 
```

Comment:
please change import sbt._ to import sbt.*

Reply:
also in build.sbt 
Seq[Def.Setting[_]] to Seq[Def.Setting[?]]
sbt version could be 1.8.3